Zip2address/README.md

```diff
@@ -1,61 +0,0 @@
-Japanese Postcode Address converter
-================
```

Comment:
Did this Readme become obsolete?

Reply:
I think... it should be rewritten completely.